compiler/rustc_typeck/src/check/fn_ctxt/checks.rs

```diff
@@ -27,6 +27,11 @@ use crate::structured_errors::StructuredDiagnostic;
 use std::iter;
 use std::slice;
 
+enum TupleAsFnArg<'hir> {
```

Comment:
I realized that the name I suggested is actually a bit backwards 😆 

What do you think of this instead?

```suggestion
enum FnArgsAsTuple<'hir> {
```

Reply:
My bad, should've read it too! 😆